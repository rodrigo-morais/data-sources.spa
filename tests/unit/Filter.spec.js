import { shallowMount } from '@vue/test-utils'
import { createRenderer } from 'vue-server-renderer'
import SearchFilter from '@/components/SearchArea/Filter/index.vue'

describe('SearchFilter.vue', () => {
  it('renders according to design', () => {
    const renderer = createRenderer()
    const wrapper = shallowMount(SearchFilter)

    renderer.renderToString(wrapper.vm, (err, str) => {
      if (err) throw new Error(err)
      expect(str).toMatchSnapshot()
    })
  })
})
