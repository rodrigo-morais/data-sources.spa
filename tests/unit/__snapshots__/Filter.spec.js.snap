// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchFilter.vue renders according to design 1`] = `
<div>
  <SearchTextBox-stub></SearchTextBox-stub>
</div>
`;
